### ISSUE TITLE
Is technique H101 sufficient for SC 1.3.6?

### ISSUE THREAD

**Issue by dafydd61** [w3c/wcag#3665]
At https://www.w3.org/WAI/WCAG22/Techniques/html/H101:

Should 1.3.6: Identify Purpose (Sufficient) be added to "This technique relates to"?

This comes out of the following A11y Slack thread:
https://web-a11y.slack.com/archives/C7KTV5CBA/p1706548045310479

@scottaohara

**Comment by scottaohara** [comment]
thanks for logging.  will assign myself to review / work on this issue as needed.
